modules/http/http.c

```diff
@@ -108,6 +108,8 @@ _setup_static_options_in_curl(HTTPDestinationWorker *self)
 
   if (owner->ca_file)
```

Comment:
I guess there is no need for the if, simple: `curl_easy_setopt(self->curl, CURLOPT_CAINFO, owner->ca_file);` would do the same.

Reply:
indeed! will change it.
(background: originally, I set an empty string instead of `NULL`, but it did not work...  as we need to set an explicit `NULL`, the `else` is superfluous)